root/layout/components/Footer.js

```diff
@@ -36,7 +36,7 @@ const Footer = ({$c, ...props}) => {
         ]}
 
         {!!DBDefs.GIT_BRANCH && [
```

Comment:
Alternatively, if we remove the array here and use a fragment (I guess this was written before we had those), we wouldn't need keys.

Reply:
Done in https://github.com/metabrainz/musicbrainz-server/pull/717/commits/993ed78551d2d25d056139f4c59ec8e4f4d2e0b3